aiven/client/cliarg.py

```diff
@@ -46,7 +49,7 @@ def user_config_json():
     def wrapper(fun):
         arg(
             "--user-config-json",
-            default="{}",
+            default={},
```

Comment:
Imho something line `None` would make a better default here. Now the function takes in basically `dict` or `str` whereas `Optional[str]` would maybe be more consistent. I know there's no typing here but still.

Reply:
Fixed.